exercise/core/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Model for recipes"""
+
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=255, blank=True)
```

Comment:
You could even use a [TestField](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.TextField)